app/components/forms/form-switch.js

```diff
@@ -0,0 +1,35 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import abstractMethod from 'travis/utils/abstract-method';
+
+export default Component.extend({
+  tagName: 'button',
+  classNames: ['travis-switch', 'switch', 'inline-block'],
+  classNameBindings: ['checked:active', 'disabled:disabled'],
+  attributeBindings: ['aria-checked', 'role', 'type'],
+
+  type: 'button',
+  role: 'switch',
+  checked: false,
+  disabled: false,
+
+  onInit() {},
+  onChange() {},
+
+  onToggle: abstractMethod('onToggle'),
+
+  didInsertElement() {
+    this._super(...arguments);
+    this.onInit(this.elementId);
+  },
+
+  'aria-checked': computed('checked', function () {
```

Comment:
It would be better to give the property a valid Javascript name, like `ariaChecked`, so that it could be re-used, and just bind it in `attributeBindings`:
```js
attributeBindings: ['ariaChecked:aria-checked', 'role', 'type'],
```

Reply:
1 - 1 from travis switch